administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -181,11 +181,15 @@
 	</div>
 </form>
 <?php
+JHtml::_('jquery.framework', false);
 $document = JFactory::getDocument();
 $document->addScriptDeclaration('
-function copytext() {
-textBox = document.getElementById("articletitle");
-message = document.getElementById("menutitle");
-message.value = textBox.value;
+$(document).ready(function () {
+$("input[name=title]").keyup(function () {
+var a = $("input[name=title]").val();
+var b = a;
+$("input[name=menutitle]").val(b);
+});
```

Comment:
Btw the placement of jquery code is correct i.e in edit.php file after the form code?